packages/liquidator/test/Liquidator.js

```diff
@@ -1044,6 +1049,12 @@ contract("Liquidator.js", function(accounts) {
           // This places their position with a CR of: 115 / (100 * 1) * 100 = 115%. This is below the CR threshold.
           await emp.requestWithdrawal({ rawValue: convert("10") }, { from: sponsor1 });
 
+          // Advance time to the defenseActivationPercent to see if the WDF would trigger.
+          let sponsor1Positions = await emp.positions(sponsor1);
+          const withdrawLiveness = empProps.withdrawLiveness.toNumber();
+          let nextTime = Math.ceil(Number(sponsor1Positions.withdrawalRequestPassTimestamp) - withdrawLiveness * 0.5);
+          await emp.setCurrentTime(nextTime);
```

Comment:
nice catch here, yea this test probably would not have worked as intended

Reply:
Yeah well it works but its a big of a false positive

Reply:
For others: without this line here, the WDF would not trigger even if the bot couldn't fully liquidate the position because not enough time has passed (e.g. `defenseActivationPercent`).